<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerWidget</class>
 <widget class="QWidget" name="PlayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>441</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="selectDirBtn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件夹</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>180</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="rightPart">
    <item>
     <widget class="QPushButton" name="playChosenBtn">
      <property name="text">
       <string>播放</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delFromListBtn">
      <property name="text">
       <string>移出歌单</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="choseListBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="addToListBtn">
        <property name="text">
         <string>加入歌单</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="delListBtn">
        <property name="text">
         <string>删除歌单</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="updInfoBtn">
      <property name="text">
       <string>修改信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delChosenBtn">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>243</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="topPart">
    <item>
     <widget class="QLineEdit" name="searchLine"/>
    </item>
    <item>
     <widget class="QPushButton" name="searchBtn">
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
